packages/aggregate-client/src/types.ts

```diff
@@ -70,19 +69,12 @@ export interface Connectable {
  */
 export type CARLink = Link<unknown, typeof CAR.codec.code>
 
-export type OfferSrc = ToString<URL>
-
-/**
- * [Piece CID](https://spec.filecoin.io/systems/filecoin_files/piece/) of some
- * content.
- */
-export type PieceCID = ReturnType<typeof CommP.toCID>
-
 /**
+ * Filecoin piece proof that can be used to derivew `PieceInfo`.
  * [Piece](https://spec.filecoin.io/systems/filecoin_files/piece/) information
  * for this CAR file.
  */
 export interface Piece {
-  link: PieceCID
-  size: number
+  link: PieceLink
+  height: number
 }
```

Comment:
Looks like we need another type [along with `Piece` and `PieceInfo`](https://github.com/web3-storage/data-segment/blob/61ee74d4ebc5c8eec8b5d4dcea2b46e844fe1dc8/src/api.ts#L119-L165) that we can re-export here.

Reply:
https://github.com/web3-storage/data-segment/pull/17